pkgs/development/python-modules/patchpy/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  rye,
+  hatchling,
+  more-itertools,
+  pytestCheckHook
+}:
+
+buildPythonPackage {
+  pname = "patchpy";
+  version = "0-unstable-2024-05-26";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "MatthewScholefield";
+    repo = "patchpy";
+    rev = "dcd673a0c2bf3f38e2b6b02e1a968be4f4d6c267";
+    hash = "sha256-q5AaaNcf5HQ1OurzebuPiMTR44NMPxgSEiFzmcZqrFY=";
+  };
+
+  build-system = [ setuptools hatchling ];
```

```suggestion
  build-system = [ hatchling ];
```
https://github.com/MatthewScholefield/patchpy/blob/dcd673a0c2bf3f38e2b6b02e1a968be4f4d6c267/pyproject.toml#L16